Makefile.core.mk

```diff
@@ -437,8 +437,8 @@ security-test:
 common-test: build
```

Comment:
this target isn't run in CI, we call common-racetest. Now that I am thinking about it, we should probably just remove the racetest targets and make these ones pass `-race`

Reply:
Do you want me to do it in this PR?

Reply:
I was thinking this change was not tested as a result of this. If it is tested still we can do it separately

Reply:
Yes, I ran `make common-test` by hand and made sure everything is passing.